images/php/cli/ssh_config

```diff
@@ -2,4 +2,6 @@ Host *
   StrictHostKeyChecking no
   UserKnownHostsFile /dev/null
   IdentityFile /home/.ssh/lagoon_cli.key
-  IdentityFile /home/.ssh/key
\ No newline at end of file
+  IdentityFile /home/.ssh/key
+  ServerAliveInterval 60
+  ServerAliveCountMax 1440 # max keepalive of 24h
```

Comment:
```suggestion
  ServerAliveCountMax 1440
```
ssh_config does not support comments at the end of a config line, and fails with
```
/etc/ssh/ssh_config line 7: garbage at end of line; "#".```